src/raster/r.survey/r.survey.html

```diff
@@ -0,0 +1,290 @@
+<dd>Verbose module output</dd>
+<dt><b>--quiet</b></dt>
+<dd>Quiet module output</dd>
+<dt><b>--ui</b></dt>
+<dd>Force launching GUI dialog</dd>
+</dl>
+</div>
+
+<div id="parameters">
+<h3>Parameters:</h3>
+<dl>
+<dt><b>points</b>=<em>name</em>&nbsp;<b>[required]</b></dt>
+<dd>Name of input vector map</dd>
+<dd>Name of the input points map  (representing the survey location). It can not be a 3D vector layer.</dd>
+
+<dt><b>dem</b>=<em>name</em>&nbsp;<b>[required]</b></dt>
+<dd>Name of the input DEM layer</dd>
+
+<dt><b>output</b>=<em>name</em>&nbsp;<b>[required]</b></dt>
+<dd>Prefix for the output visibility layers</dd>
+
+<dt><b>maxdist</b>=<em>float</em>&nbsp;<b>[required]</b></dt>
+<dd>max distance from the input points</dd>
+<dd>Default: <em>1000</em></dd>
+
+<dt><b>treesmap</b>=<em>name</em></dt>
+<dd>Name of input vector map</dd>
+<dd>Name of the vector layer representing the forested areas</dd>
+
+<dt><b>buildingsmap</b>=<em>name</em></dt>
+<dd>Name of input vector map</dd>
+<dd>Name of the vector layer representing the buildings</dd>
+
+<dt><b>obs_heigh</b>=<em>double</em>&nbsp;<b>[required]</b></dt>
+<dd>observer_elevation</dd>
+<dd>Default: <em>1.75</em></dd>
+
+<dt><b>treesheigh</b>=<em>string</em></dt>
+<dd>field of the attribute table containing information about threes heigh</dd>
+
+<dt><b>treesheighdeviation</b>=<em>string</em></dt>
+<dd>field of the attribute table containing information about standard deviation value relate to the heigh</dd>
+
+<dt><b>buildingsheigh</b>=<em>string</em></dt>
+<dd>field of the attribute table containing information about buildings heigh</dd>
+
+<dt><b>obsabselev</b>=<em>string</em></dt>
+<dd>field of the attribute table containing information about observer absolute elevation above the same datum used by the DEM (e.g. geodetic elevation)</dd>
+
+<dt><b>layer</b>=<em>string</em></dt>
+<dd>layer of the attribute table of the point map (can be usefull for obsabselev option)</dd>
+<dd>Default: <em>1</em></dd>
+
+<dt><b>viewangle_threshold</b>=<em>double</em>&nbsp;<b>[required]</b></dt>
+<dd>cut the output layers at a given threshold value (between 90 and 180 degrees)</dd>
+<dd>Default: <em>90.01</em></dd>
+
+<dt><b>object_radius</b>=<em>double</em></dt>
+<dd>radius of the surveyed object in unit of map (default is half the DEM resolution)</dd>
+
+<dt><b>nprocs</b>=<em>integer</em></dt>
+<dd>Number of processes</dd>
+<dd>Default: <em>1</em></dd>
+
+<dt><b>memory</b>=<em>integer</em></dt>
+<dd>Amount of memory to use in MB (for r.viewshed analysis)</dd>
+<dd>Default: <em>500</em></dd>
+
+</dl>
+
+<h2>DESCRIPTION</h2>
+
+r.survey is aimed at earth surface analysis. It is useful for evaluating the visibility (in terms of 3D Distance, Solid Angle and orientation of the terrain respect to the line of sight) of  features lying on the terrain slope, like landslides, road cuts, minor vegetation, mining activity, geological outcrops, surface pipelines, parking areas, solar panels plants, burned areas, etc. 
```

Comment:
```suggestion
<em>r.survey</em> is aimed at earth surface analysis. It is useful for evaluating the visibility (in terms of 3D Distance, Solid Angle and orientation of the terrain respect to the line of sight) of  features lying on the terrain slope, like landslides, road cuts, minor vegetation, mining activity, geological outcrops, surface pipelines, parking areas, solar panels plants, burned areas, etc. 
```

Please follow the https://trac.osgeo.org/grass/wiki/Submitting/Docs#Markupstyleguide